storybook/storybook-react/src/Grid/Grid.Cell.stories.tsx

```diff
@@ -10,6 +10,20 @@ import { Meta, StoryObj } from '@storybook/react'
 const meta = {
   title: 'Layout/Grid Cell',
```

Comment:
If you remove this line it will be hidden from the navigation tree and the stories described in grid docs will have to suffice.

Reply:
Have you seen that working? Doesn’t change anything for me.

As indicated here, I’ve tried the following in both the docs and the react instance, with `tags: ['hidden']` in the grid cell `meta`, unfortunately to no effect either:

```js
addons.setConfig({
  theme: create(themeConfig),
  sidebar: {
    filters: {
      hidden: (item) => {
        console.log(item.title, item.tags)
        return !item.tags.includes('hidden')
      },
    },
  },
})
```

It‘s only logging the docs entries to the console – so might have something to do with the composition. @alimpens does that ring a bell for you?

I’ve just pushed it as a WIP commit.

Reply:
That is the goal yes. We discussed we wanted to remove the separation between `Grid` and `Grid.Cell` as much as possible because they are inseparable.

The problem I am trying to overcome is that both have different props, so they don’t share the same `meta` for example, and Storybook doesn’t seem to work well with one `*.stories.tsx` file defining two sets of meta’s, stories etc.

Removing `.stories` from the file name doesn’t work either. It results in the error

> Invalid value passed to the 'of' prop. The value was resolved to a 'component or unknown' type but the only types for this block are: story. (…) Did you pass a story, CSF file or meta to the 'of' prop that is not indexed, ie. is not targeted by the 'stories' globs in the main configuration?”

Reply:
Ah, you're right, that doesn't work. I think you're trying to do the same thing I did for [Search Field](https://github.com/Amsterdam/design-system/pull/892/commits/afdfbfb08f5d4e53b15d21cd2d89b95a6dc9f16a), did you look at that?

Reply:
I did, and in the end the render function helped – but I think the context is a bit different so my approach is as well. Still, it worked.

I’d like to limit the controls to the ones that the specific story pertains to. Is that worthwhile?